VideoLocker/src/main/java/org/edx/mobile/util/AppConstants.java

```diff
@@ -1,12 +1,17 @@
 package org.edx.mobile.util;
 
 
-public class AppConstants {
+public final class AppConstants {
```

Comment:
Lemme make that an enum class, which will won't allow it to be extended nor an object of it to be instantiated.